wasm/package-benchmark.sh

```diff
@@ -28,14 +28,9 @@ sed -i.bak 's/return WebAssembly.instantiate(binary, info);/return WebAssembly.i
 sed -i.bak 's/var module = new WebAssembly.Module(bytes);/var module = new WebAssembly.Module(bytes, {simdWormhole:true});/g' marian-decoder.js
 echo "SUCCESS"
 
-echo "Polyfill the fallback integer (8-bit) gemm implementation from the main module"
-sed -i.bak 's/asmLibraryArg,/asmLibraryArg,"wasm_gemm":{\
-    "int8_prepare_a": (...a) => Module["asm"].int8PrepareAFallback(...a),\
-    "int8_prepare_b": (...a) => Module["asm"].int8PrepareBFallback(...a),\
-    "int8_prepare_b_from_transposed": (...a) => Module["asm"].int8PrepareBFromTransposedFallback(...a),\
-    "int8_prepare_b_from_quantized_transposed": (...a) => Module["asm"].int8PrepareBFromQuantizedTransposedFallback(...a),\
-    "int8_prepare_bias": (...a) => Module["asm"].int8PrepareBiasFallback(...a),\
-    "int8_multiply_and_add_bias": (...a) => Module["asm"].int8MultiplyAndAddBiasFallback(...a),\
-    "int8_select_columns_of_b": (...a) => Module["asm"].int8SelectColumnsOfBFallback(...a),\
-    },/g' marian-decoder.js
+echo "Importing integer (8-bit) gemm implementation"
+SCRIPT_ABSOLUTE_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
Is "$(dirname "$0")" sufficient for these purposes?

Reply:
`"$(dirname "$0")"` gives the relative path and I thought of working with the absolute path (although you can argue that relative path might be enough in this case). It is just a matter of choice. I am fine with anything here.